src/service/git/git-cli.ts

```diff
@@ -110,10 +110,14 @@ export default class GitCLIService {
    * @param cwd repository in which the sha should be cherry picked to
    * @param sha commit sha
    */
-  async cherryPick(cwd: string, sha: string, strategy = "recursive", strategyOption = "theirs"): Promise<void> {
+  async cherryPick(cwd: string, sha: string, strategy = "recursive", strategyOption = "theirs", cherryPickOptions: string | undefined): Promise<void> {
     this.logger.info(`Cherry picking ${sha}`);
     
-    const options = ["cherry-pick", "-m", "1", `--strategy=${strategy}`, `--strategy-option=${strategyOption}`, sha];
+    let options = ["cherry-pick", "-m", "1", `--strategy=${strategy}`, `--strategy-option=${strategyOption}`];
+    if (cherryPickOptions !== undefined) {
+      options.concat(cherryPickOptions.split(" "));
```

Comment:
```suggestion
      options = options.concat(cherryPickOptions.split(" "));
```

Reply:
:facepalm: 

Thanks for catching this, I was debugging it manually and you fixed it!